dev-libs/tomsfastmath/tomsfastmath-0.13.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="TomsFastMath is a fast public domain large integer arithmetic library"
+HOMEPAGE="https://www.libtom.net/TomsFastMath/ https://github.com/libtom/tomsfastmath"
+SRC_URI="https://github.com/libtom/tomsfastmath/releases/download/v${PV}/tfm-${PV}.tar.xz"
+LICENSE="Unlicense"
+
+SLOT="0/1"
```

Comment:
Can you please add a comment here stating what the subslot `1` refers to (so that in the future, other maintainers will know when to bump it)?

Reply:
To answer my own question (but you should still add a comment): it's the major component in `VERSION=1:0:0` in the makefile.shared, which gets used as the soname.

Reply:
👍